docs/design-docs/08-package-variant.md

```diff
@@ -292,24 +300,51 @@ The package variant set controller is designed to fill this common need. This
 controller consumes PackageVariantSet resources, and outputs PackageVariant
 resources. The PackageVariantSet defines:
 - the upstream package
-- a list of targets
-- rules for generating one PackageVariant per target
+- targeting criteria
+- a template for generating one PackageVariant per target
 
 Three types of targeting are supported:
-- An explicit list of repository and package names
-- A label selector for Repository objects along with a way to generate package
-  names from those objects
-- An arbitrary object selector, along with ways to generate repository and
-  package names from those objects
-
-*Figure 5* shows an example of creating PackageVariant resources based upon the
-explicitly list of repository and package names.
+- An explicit list of repositories and package names
+- A label selector for Repository objects
+- An arbitrary object selector
 
-![Figure 5: List of Targets](packagevariantset-target-package.png)
+Rules for generating a PackageVariant are associated with a list of targets
+using a template. That template can have explicit values for various
```

Comment:
I'm not sure about the use of the word `template` here. I was at first thinking it would be similar to the PodSpec template in a Deployment resource, but this is somewhat different. At first this seems more like an overlay.

Reply:
Let's see what you think once I publish this. It's a lot like that.

Reply:
Ok, take a look at the complete API as published, see what you think. I think `template` is appropriate.